src/sentry/lang/javascript/processor.py

```diff
@@ -409,11 +409,13 @@ def fetch_file(url, project=None, release=None, allow_scraping=True):
                         error = {
                             'type': EventError.JS_FETCH_TIMEOUT,
                             'url': expose_url(url),
+                            'timeout': settings.SENTRY_SOURCE_FETCH_TIMEOUT,
                         }
                     elif isinstance(exc, OverflowError):
                         error = {
                             'type': EventError.JS_TOO_LARGE,
                             'url': expose_url(url),
+                            'max_size': settings.SENTRY_SOURCE_FETCH_MAX_SIZE,
```

Comment:
You could do this math here. https://github.com/getsentry/sentry/pull/4359/files#r83334946